v1.0/docs/api-reference/definitions.md

```diff
@@ -1,5 +0,0 @@
----
```

Comment:
Also did you manually delete docs/api-reference/*.html?
The tool should have produced those files.